src/main/java/org/apache/sling/junit/impl/servlet/JUnitResultRenderer.java

```diff
@@ -23,21 +23,24 @@
 
 import javax.servlet.http.HttpServletResponse;
 
-import org.apache.felix.scr.annotations.Component;
-import org.apache.felix.scr.annotations.Service;
 import org.apache.sling.junit.Renderer;
 import org.apache.sling.junit.RendererFactory;
 import org.apache.sling.junit.TestSelector;
 import org.junit.runner.Result;
 import org.junit.runner.notification.RunListener;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.ServiceScope;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /** Renderer for Sling JUnit server-side testing, which
  *  renders the serialized JUnit Result object.
  */
-@Component
-@Service
+@Component(
+    service = Renderer.class,
+    immediate = false,
+    scope = ServiceScope.BUNDLE
```

Comment:
scope=bundle changes the semantics, IMHO a global singleton should be fine (which is the default scope)

Reply:
Removed.